src/java.base/share/classes/java/lang/String.java

```diff
@@ -3934,12 +3943,12 @@ public String stripTrailing() {
     }
 
     /**
-     * Returns {@code true} if the string is empty or contains only
```

Comment:
This method seems like a good candidate to use `{@return ...}`.

Reply:
I'll give this a try.

Reply:
Thanks, this is a nice simplification.